db/patterns/zypmedia.eno

```diff
@@ -1,6 +1,6 @@
 name: ZypMedia
 category: advertising
-website_url: http://www.zypmedia.com/
+website_url: https://www.zypmedia.com/opt-out
```

Comment:
Ah, now I understand where the "opt-out" link comes from. This is a special case as the company has been acquired.

I would recommend to leave the old URL as is for now (not change in this PR). I split it out in a separate issue since it requires more research: https://github.com/ghostery/trackerdb/issues/119